docs/source/intervention_models/maternal_anemia_intervention/index.rst

```diff
@@ -0,0 +1,248 @@
+   intervention is not applicable; 
+ - the impact at each tier on maternal anemia in terms of averted deaths and DALYs 
+   if we assume complete pyramid of interventions are applied.
+
+.. list-table:: Tiered Intervention
+  :header-rows: 1
+
+  * - Tier
+    - Intervention
+    - Product name
+    - Product definition
+    - Target population
+    - Note
+  * - 1
+    - Large-scale food fortification
+    - DFS
+    - 
+    - Women of reproductive age (15 to 49 years)
+    - 
+  * - 2
+    - Supplementation
+    - BEP
+    - 
+    - PLW at ANC service with BMI less than 18.5
+    - 
+  * - 2
+    - Supplementation
+    - MMS
+    - 
+    - PLW at ANC service with BMI greater than or equal to 18.5
+    - 
+  * - 3
+    - Treatment
+    - IV iron
+    - 
+    - Pregnant women in 2nd/3rd trimester with moderate/severe anemia referred 
+      from ANC to higher level facility
+    - 
+
+.. todo::
+
+  Fill out the table above with intervention product definition.
+
+.. list-table:: Affected Outcomes
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Population
+    - Note
+  * - Population mean hemoglobin
+    - Additive
+    - non-PLW
+    - Effects are not stratified by intervention tiers
+  * - Population mean hemoglobin
+    - Additive and multiplicative combined
+    - PLW
+    - Effects are stratified by intervention tiers 
+
+Baseline Coverage Data
+++++++++++++++++++++++
+
+.. note::
+
+  No existing coverage of each intervention tier
+
+
+.. list-table:: Baseline coverage data
+  :widths: 10 10 10 10 10
+  :header-rows: 1
+
+  * - Location
+    - Subpopulation
+    - Coverage parameter
+    - Value
+    - Note
+  * - Bangladesh
+    - 
+    - 
+    - 
+    - 
+  * - Malawi
+    - 
+    - 
+    - 
+    - 
+  * - Nigeria
+    - 
+    - 
+    - 
+    - 
+  * - Pakistan
+    - 
+    - 
+    - 
+    - 
+
+
+Vivarium Modeling Strategy
+--------------------------
+How to estimate maternal anemia burden in Vivarium:
+ 1. Construct population hemoglobin distributions for each location-/age- group;
+ 2. Apply pregnancy adjustment factor to hemoglobin distributions for pregnant 
+    and general populations;
+ 3. Modify hemoglobin distributions based on intervention risk effects;
+ 4. Calculate maternal anemia burden, measured by deaths, YLDS, and YLLs.
+
+Hemoglobin
+++++++++++
+
+.. list-table:: Modeled Outcomes
+  :widths: 10 10 10 10 10 10 10
+  :header-rows: 1
+
+  * - Outcome
+    - Outcome type
+    - Outcome ID
+    - Affected measure
+    - Effect size measure
+    - Effect size
+    - Note
+  * - Hemoglobin
+    - GBD risk
+    - rei_92
+    - Exposure prevalence
+    - Relative risk
+    - Detailed in table `Effect Size`
+    - 
+
+.. list-table:: Restrictions
+  :widths: 10 10 10
+  :header-rows: 1
+
+  * - Restriction
+    - Value
+    - Note
+  * - Male only
+    - False
+    - 
+  * - Female only
+    - True
+    - 
+  * - Age group start
+    - 15 years
+    - 
+  * - Age group end
+    - 49 years
+    - 
```

Comment:
I think GBD definition of reproductive age is actually 10 to 54. We trimmed it for this project by request of BMGF, but we probably want to keep it consistent with GBD otherwise.